pipeline/metadata/5.3.yaml

```diff
@@ -1075,3 +1075,33 @@
     - ibmc
     - rgw
     - stage-2
+
+- name: "test cephadm cluster bootstrap with mutiple public network"
+  suite: "suites/pacific/cephadm/customer_bz/tier-2_multiple_subnet_openstack_only.yaml"
+  global-conf: "conf/pacific/cephadm/5-node-cluster-multiple-subnet-openstack-only.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+  metadata:
+    - schedule
+    - tier-2
+    - openstack-only
+    - dmfg
+    - stage-7
+
+- name: "test to validate dashboard alerts"
+  suite: "suites/pacific/dashboard/tier-1_dashboard_alerts.yaml"
+  global-conf: "conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml"
```

Comment:
Hey @manasagowri and @sunilangadi2 ,

please check whether `conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml` file exists?
I was not able to find this file in repository.

![image](https://user-images.githubusercontent.com/31158377/188904309-40f7c32a-f64e-4ce8-b807-f8f1f8da6d01.png)

Reply:
+1

Reply:
Sorry @sunilkumarn417 it got missed with numeric 5,
Thanks for the cross check.
updated the proper conf file which exist in ceph-ci
Test results for the same conf:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-QZVCNK/
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BNY9NS